webapp/platform/types/src/config.ts

```diff
@@ -74,6 +74,7 @@ export type ClientConfig = {
     EnableGifPicker: string;
     EnableGuestAccounts: string;
     EnableIncomingWebhooks: string;
+    EnableRenderEmoticonsAsEmoji: string;
```

Comment:
These changes would also need to be reverted if we remove the server-wide setting

Reply:
Done.